app/models/firm_office.rb

```diff
@@ -1,2 +1,4 @@
 class FirmOffice < ApplicationRecord
+  has_one :prior_authority_applications, dependent: :destroy
```

Comment:
wouldn't that be a `has_many`? - using the plural seems odd, something else going on here?

Reply:
See above comments from Patrick https://github.com/ministryofjustice/laa-submit-crime-forms/pull/1372#discussion_r1890405532

Reply:
i'll go with the Patrick ESP but surely a `firm_office` does have multiple claims though? and if we wanted to redact/anonymise we could be be scoping claims within that has_many

Reply:
I think the model might be slightly incorrectly named but he is right. It's the output of what you put into the "Firm Details" step and while yes in theory that should be wrong that is how the system works.